automation-roles/99-generic/openshift/openshift-download-client/tasks/main.yml

```diff
@@ -1,50 +0,0 @@
----
```

Comment:
I would like to keep the openshift-download-client role, just in case it is useful in the future